controllers/controller.js

```diff
@@ -0,0 +1,9 @@
+const {getAllTopics} = require("../model/model")
+module.exports = {getTopics}
+
+function getTopics(request, response){
+    getAllTopics()
```

Comment:
Having these function names so similar could be confusing. I think getTopics is appropriate for your controller function but your model would be better with something like selectAllTopics